xc/common/cmake/arch_define.cmake

```diff
@@ -151,12 +151,11 @@ function(ADD_XC_ARCH_DEFINE)
         \${FASM_TO_BIT_EXTRA_ARGS}"
     BIT_TO_V bitread
     BIT_TO_V_CMD "${CMAKE_COMMAND} -E env \
-    PYTHONPATH=${symbiflow-arch-defs_BINARY_DIR}/env/conda/lib/python3.7/site-packages:${PRJRAY_DIR}:${PRJRAY_DIR}/third_party/fasm:${symbiflow-arch-defs_SOURCE_DIR}/xc/${FAMILY}:${symbiflow-arch-defs_SOURCE_DIR}/utils \
+    PYTHONPATH=${symbiflow-arch-defs_BINARY_DIR}/env/conda/lib/python3.7/site-packages:${PRJRAY_DIR}:${PRJRAY_DIR}/third_party/fasm:${symbiflow-arch-defs_SOURCE_DIR}/utils \
         \${PYTHON3} -mfasm2bels \
         \${BIT_TO_V_EXTRA_ARGS} \
         --db_root ${PRJRAY_DB_DIR}/${PRJRAY_ARCH} \
         --rr_graph \${OUT_RRBIN_REAL_LOCATION} \
-        --vpr_capnp_schema_dir ${VPR_CAPNP_SCHEMA_DIR} \
```

Comment:
How did you remove the use of the schema dir?

Reply:
I have added it in the fasm2bels repository. IIRC I had troubles in perf-tools to get the capnp schemas as they were within the arch-defs repository.

Actually, as this would need to be adjusted in the fasm2bels repo, and moved to the `github.com/SymbiFlow/symbiflow-rr-graph` IMO

Reply:
> I have added it in the fasm2bels repository. IIRC I had troubles in perf-tools to get the capnp schemas as they were within the arch-defs repository.

I believe the VPR conda package has been packing them?

Reply:
And IMO as another cleanup step we should also make use of the external rr-graph library as well

Reply:
> I believe the VPR conda package has been packing them?

Oh, right. They end up in the conda_prefix base dir, I think it would be better to have the capnp dir in the `share/vpr/` one. And maybe access to it through sth similar as for `prjxray-config`, that returns the location of the vpr schema dir. 

E.g. 
```
$ vpr-schemas 
path/to/schema/dir
```

Reply:
PR is green, this is still an issue

Reply:
Fixed. I still need to address https://github.com/SymbiFlow/symbiflow-xc-fasm2bels/issues/7 so that we can completely avoid having capnp schemas within fasm2bels and feed them through the parameter.

For now symbiflow-xc-fasm2bels has the override parameter for the schema dir. I see XC7 vendor tools CI went green, need to wait on 200T.

@litghost Would you prefer having https://github.com/SymbiFlow/symbiflow-xc-fasm2bels/issues/7 fixed before merging this PR?

Reply:
> @litghost Would you prefer having [SymbiFlow/symbiflow-xc-fasm2bels#7](https://github.com/SymbiFlow/symbiflow-xc-fasm2bels/issues/7) fixed before merging this PR?

It's a requirement.  If VPR's capnp changes (which is something we are planning on doing), your copy paste solution fails immediately.